.github/scripts/action_tools.py

```diff
@@ -224,6 +224,41 @@ def evaluate(models: List[str], datasets: List[str], workspace: str):
         add_summary(output_csv)
 
 
+def evaluate_new(models: List[str], datasets: List[str], workspace: str):
```

Comment:
what does "new" refer to?

Reply:
sorry. I'm tring to use one task to reduce timecost, so add a new method. This code should not in this pr. 
Thanks! I have remove it.